examples/scalajs-react/src/main/scala/todomvc/CTodoList.scala

```diff
@@ -70,62 +83,64 @@ object CTodoList {
           <.input(
             ^.className     := "new-todo",
             ^.placeholder   := "What needs to be done?",
-            ^.onKeyDown  ~~>? handleNewTodoKeyDown _,
+            ^.onKeyDown   ==>? cbs.value().handleNewTodoKeyDown,
```

Comment:
Yeah, i admit it looks a bit bad, but it's nothing like calling `Option.get` for example - it's completely safe. What it is meant to signal to the reader is that calling `value()` might trigger a new `Callbacks` instance to be created if, in this case, `Props` has changed in a meaningful way since last time it was accessed. I think i will add a comment in the source to make it clearer.